pkg/virt-controller/watch/vm.go

```diff
@@ -2651,18 +2660,15 @@ func (c *VMController) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachin
 }
 
 func (c *VMController) handleDynamicIfaceRequestOnVMI(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) error {
```

Comment:
Done

Reply:
I do not understand how you compared the copies, not that it is the main issue here.
There are no deep copies in this function, but a new copy was added to the caller.

Anyway, I still think we should get rid of this function now. After extracting from it almost everything, it does not make sense to keep it.
But I will not insist, I had a PR already to refactor these stuff anyway, I can fix in a follow up myself.

Reply:
> I do not understand how you compared the copies, not that it is the main issue here.
There are no deep copies in this function, but a new copy was added to the caller.

Yes, I counted cloned objects (using deep copy), I didn't count function arguments because they are pointers, there is no actual copy per my understanding.

Done.